content_sync/pipelines/definitions/concourse/site-pipeline.yml

```diff
@@ -22,18 +16,8 @@ resources:
     source:
       uri: https://github.com/mitodl/ocw-hugo-projects.git
       branch: ((ocw-hugo-projects-branch))
-  - name: ocw-site
-    type: rclone
```

Comment:
ok, my thinking was that since I have to do a conditional s3 sync in a bash shell, I might as well do all the s3 syncing there.  But I can split the other s3 sync commands out and back to rclone if you prefer.

Reply:
@tmacey now I remember, rclone doesn't seem to handle direct s3 source to s3 destination syncing, so that's 2 out of 3 syncs to be done in a bash shell, which is why I decided it made more sense to just run all 3 syncs together in a shell.

Reply:
Yeah, it's not a requirement to use a resource, but it does add some niceties to the overall UX with Concourse. Also, we can make our own resource that does exactly what we need. I don't want to be a blocker for this PR, but something that we should track so we don't lose sight of it.

Reply:
I just made an issue for that in ol-infrastructure: https://github.com/mitodl/ol-infrastructure/issues/250